app/plugins/Files/js/loaders.js

```diff
@@ -76,7 +89,9 @@ function uploadDir(dirPath, nickname) {
 					notify('Cannot read ' + file, 'error');
 					return;
 				}
-				if (!isUnixHiddenPath(filePath) && stats.isFile()) {
+				if (file.slice(-4) === '.sia') {
+					loadDotSia(filePath);
```

Comment:
I would think so. Or just modify `Add .sia File` to allow selecting multiple files.